neurodamus/utils/memory.py

```diff
@@ -361,8 +366,14 @@ def display_total(self):
         logging.info("| {:<40s} | {:12.1f} |".format(f"Overhead (ranks={MPI.size})", full_overhead))
         logging.info("| {:<40s} | {:12.1f} |".format("Cells", self.cell_memory_total))
         logging.info("| {:<40s} | {:12.1f} |".format("Synapses", self.synapse_memory_total))
+        self.simulation_estimate = (self.cell_memory_total
```

Comment:
I had understood that 2.5x was the factor among the totals. E.g. if we had estimated using 10 we were seeing 25 being used during simulation. If that's the case 25 should me considered the grand total, not exclusively the simulation, and hence not summed to the base 10.

Reply:
I double checked just to make sure about it and no, the simulation memory usage data I took is separate from the cell + synapse instantiation and therefore should be added to it and does not represent the total.

By re-running the test for this purposes I noticed a significantly smaller memory footprint for the simulation than what we've seen in the past, possibly due to memory optimization of the last months.

I will leave the 2.5x factor for now but we might want to consider to lower it a bit in the future.